### ISSUE TITLE
if default 27 inv slots fool, no items will be picked up.

### ISSUE THREAD

**Issue by reapersremorse** [Funwayguy/InfiniteInvo#42]
i understand why this may be hard to fix but it would be awesome if it was fixed.

blocks only fill 1 more slot than the normal vanilla 27 slots, magnets do not help either. 

although if it isn't a bother you may be able to put a feature in that causes picked up items would go into the very last available slot instead of the first. if this were so then there would be no problem. (i would probably set this as a config option considering some people would hate it :/ )

thanks for taking the time to read these and for making an amazing mod
have a nice day

**Comment by Funwayguy** [comment]
Last time I checked this worked just fine. Must have broken somewhere along the way

**Comment by reapersremorse** [comment]
I was using version 1.0.37 updated 20 days ago
Downloaded from drone.io

other inventory related mods
Using latest nei
Latest tinkers
Latest baubles

I cant find anything in the log

**Comment by Funwayguy** [comment]
Just tested this now with the current released version (no other mods) and couldn't reproduce it. Pickups work just fine for me when the all vanilla slots have been filled. I'll test it with the other mods later and see which one of those could be responsible.

**Comment by reapersremorse** [comment]
Thank you and sorry for botgering you.

**Comment by Funwayguy** [comment]
If you're still having this issue I think you should refer this issue to the baubles developer. All the other mods I've tested and work just fine with InfiniteInvo. I suspect baubles is intentionally preventing pickups from entering additional slots because of it's own extras.

**Comment by reapersremorse** [comment]
That could be true but i thought baubles was a seperate inventory all
together, which would make it impossible to have things be picked up and
added to one of the floor slots, and the items which go into the baubles
slots have to have either a metadata (or something) saying its a ring,
amulet or sash.

Azanor made baubles so ill post on his get soon, but after playing around,
i think its nei from chickenbones that has the issue, magnets do not allow
u to pick up more than 27 stacks of utems but walking on them sometimes does

**Comment by Mewtwoy** [comment]
The issue is NEI, after 27 slots, magnet stops working
thus, you have to move at vanilla range to pickup items

**Comment by Funwayguy** [comment]
Hmm... I see, Magnet mode doesn't work for slots outside 27

**Comment by reapersremorse** [comment]
Yes i was just testing this, it also does the same with the magn3ts from
other mods like botania and such

**Comment by Funwayguy** [comment]
Just checked NEI's code. Magnet mode is hardcoded to the vanilla inventory size so there isn't anything I can do about that from my end. I'll check botania but if they are using the Code Chicken Library then they will inherit the same problem.

**Comment by reapersremorse** [comment]
Ahh
Do u think chickenbones could add something to the library to detect how
many slots infinvo has enabled?

**Comment by Funwayguy** [comment]
Well all inventories in Minecraft have a built in method called 'getSizeInventory()' so it's not like you'd have to custom program anything to support it. InfiniteInvo will happily give you the correct size if you call that method (like most mods should instead of assuming it never changes)

**Comment by reapersremorse** [comment]
Fair point, im not a modder nor do i know java. I wish i could learn, many
ideas stay locked away in my head (some may be considered op but still fun)

Going off of what you have said though, if nei's magnet mode was
getsizeinventory instead of hard coding 27 slots, then the issue would most
likely be fixed??

**Comment by Funwayguy** [comment]
Precisely so.

I can't seem to find anything that says Botania's magnet ring checks inventory space at all.

**Comment by reapersremorse** [comment]
Im going to check again, i may have had a botania magnet and nei magnet
mode enabled.

**Comment by reapersremorse** [comment]
Only nei magnet mode has the problem, i tried the botania and ender io as
well as a few others, while the magnet mode is on hiwever it seemed to
enhibit the other magnets but not all the time. Long story short ill be
useing bauble items or blood magic for my magnet needs

**Comment by reapersremorse** [comment]
Unfortunately though, spawning items into extra slots is not allowed. You
must middle click a stack then place manually. Not a deal breaker but would
make play a bit easier. Im sure this is on chickenbones part though
considering magnet mode is hardcoded.
